### DISCUSSION TITLE
Viewport relative transforms

### DISCUSSION THREAD

**Discussion by welsmore** [leozide/leocad#773] [Ideas]
Hiya,

This is a feature that Bricksmith has, but I'm now moving to LeoCad as Bricksmith isn't being regularly updated. I'd love to see it in LeoCad!

As opposed to relative or absolute transforms, the default mode in Bricksmith is viewport related transforms. So, when using the keyboard to move bricks, pressing 'Left' always moves the brick to the 'Left' of the way you're looking at it. It might sound strange, but once you get used to it - it's *really* intuitive. 

Is there any way to add this to Leocad?

**Comment by leozide** [comment]
It's supposed to be view dependent but doesn't work very well. Make sure you have the 'Fixed direction keys' checkbox unchecked in the Preferences Dialog.

**Comment by welsmore** [comment]
I've just checked and I do. I don't think it's ever worked for me sadly :-( 

Otherwise though, I really like LeoCad! Thankyou for all the hard work.

**Reply by leozide** [reply]
I turned this into a bug, even though I thought I had one already. I'll take a look when I have time.

**Reply by nathaneltitane** [reply]
Thanks @leozide  - this has been bugging me for a very long time as well.

**Reply by welsmore** [reply]
Thankyou!
